src/cript/api/api.py

```diff
@@ -609,7 +615,7 @@ def _get_db_schema(self) -> dict:
             return self._db_schema
 
     @beartype
-    def _is_node_schema_valid(self, node_json: str, is_patch: bool = False) -> bool:
+    def _is_node_schema_valid(self, node_json: str, is_patch: bool = False, force_validation: bool = False) -> bool:
```

Comment:
I would have called it `force_schema_validation` just in case we have multiple validations in the future, but this works as well

Reply:
fair point, but I think we can refactor if need be.
Actually there other validations we still run at the moment.